src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -68,6 +87,31 @@ def build_sections(self):
                 }
             )
 
+            self._process_desc_element(node, section_title)
+
+    def _process_desc_element(self, node, title):
+        """Process `desc` element within a section."""
```

Comment:
Could we clarify about what is this `disc` object?

Reply:
The `desc` object in Sphinx is a **node used for documenting items** like functions, classes, and other domain-specific objects. This node isn’t directly added by `docutils.addnodes`; rather, it’s defined in [**Sphinx’s own `sphinx.addnodes` module**](https://github.com/sphinx-doc/sphinx/blob/master/sphinx/addnodes.py).

To index the properties and functions in API references, we need to use the desc node.